src/main/java/cat/command/AddCommand.java

```diff
@@ -0,0 +1,68 @@
+package cat.command;
+
+import java.util.HashMap;
+
+import cat.Storage;
+import cat.TaskList;
+import cat.task.Deadline;
+import cat.task.Event;
+import cat.task.Task;
+import cat.task.Todo;
+import cat.ui.Ui;
+
+/**
+ * A command to add a task to the task list.
+ */
+public class AddCommand extends Command {
+    private final Type type;
+    private final HashMap<String, String> components;
+    private final String description;
+
+    /**
+     * The possible types of tasks.
+     */
+    public enum Type {
+        Todo,
+        Deadline,
+        Event,
+    }
+
+    /**
+     * Constructs a command that adds a task to the task list.
+     *
+     * @param type       the type of the task
+     * @param components the components parsed from the user command
```

Comment:
I think we need to capitalize the first letter and end the statement with a ```.```? Similar comment applies to other Javadoc, will not repeat to avoid too much repetition!

Reply:
I don't know how I missed that guideline. Thanks for pointing that out. ^^